### ISSUE TITLE
Align namespace for SPARQL Query Results XML Format in recommendation and XML Schema

### ISSUE THREAD

**Issue by jaw111** [w3c/sparql-12#146]
## Why?

The current recommendation [1] specifies the namespace as `http://www.w3.org/2005/sparql-results#` whilst the XML schema [2] uses the namespace `http://www.w3.org/2007/SPARQL/results#`.

Due to this any XML document using the namespace in the recommendation is invalid according the schema.

[1] http://www.w3.org/TR/2013/REC-rdf-sparql-XMLres-20130321/
[2] http://www.w3.org/2007/SPARQL/result.xsd

## Previous work

[Errata for SPARQL Query Results XML Format (Second Edition)](https://www.w3.org/2013/sparql-errata#rdf-sparql-XMLres)

## Proposed solution

Change the namespace in the XML schema to `http://www.w3.org/2005/sparql-results#`.

## Considerations for backward compatibility

Could break any implementations that use the namespace `http://www.w3.org/2007/SPARQL/results#` (I don't know of any).

**Comment by afs** [comment]
This is recorded in  the SPARQl 1.1 errata:

[errata-results-xml-1](https://www.w3.org/2013/sparql-errata#errata-results-1)

**Comment by jaw111** [comment]
@afs yes, it is noted in the previous work. I raise this here as previous communication we had noted "the errata document would be input to another W3C working group", which I believe is what this community group is aiming to in the longer term.

https://lists.w3.org/Archives/Public/public-sparql-dev/2018JulSep/0026.html
